src/pages/Counter/container/CountContainer.js

```diff
@@ -0,0 +1,54 @@
+import { Component } from "react";
+
+import CounterView from "../components/CounterView";
+
+class CounterContainder extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      countValue: 0,
+      isEven: true,
+    };
+  }
+  handleDecrement = () => {
+    if (this.state.countValue > 0) {
+      this.setState((state) => ({
+        isEven:
+          state.countValue - 1 === 0 || (state.countValue - 1) % 2 === 0
+            ? true
+            : false,
+        countValue: state.countValue - 1,
+      }));
+    }
+  };
+  handleIncrement = () => {
+    this.setState((state) => ({
+      isEven: (state.countValue + 1) % 2 === 0 ? true : false,
```

Comment:
try to move logic with recalculating `isEven` property to componenDidUpdate(). The reason is that you have a lot of duplicated code fragments